lib/core/test/unit/Cardano/Wallet/Primitive/FeeSpec.hs

```diff
@@ -707,10 +709,10 @@ genSelectionFor :: NonEmpty TxOut -> Gen CoinSelection
 genSelectionFor outs = do
     let opts = CS.CoinSelectionOptions (const 100) (const $ pure ())
     utxo <- vector (NE.length outs * 3) >>= genUTxO
-    case runIdentity $ runExceptT $ largestFirst opts outs utxo of
+    withdrawal_ <- genWithdrawal
+    case runIdentity $ runExceptT $ largestFirst opts outs (Quantity withdrawal_) utxo of
```

Comment:
what about checking additionally if dusting play like expected with new player : withdrawal ? Or we are confident it is going to be fine?

Reply:
I am not sure to follow your concern on this one :thinking:  ?